services/ml/compilation_delegate_dnnl.cc

```diff
@@ -547,7 +549,8 @@ int32_t CompilationDelegateDnnl::AddReorder(const std::string& input_name,
 int32_t CompilationDelegateDnnl::AddFusedActivation(
     const std::string& input_name,
     const std::string& output_name,
-    int32_t fuse_code) {
+    int32_t fuse_code,
+    int32_t type) {
```

Comment:
Please try to combine `type` and `fuse_code`. Literally, we only need one parameter that indicates the activation type.

Reply:
Yes, I have defined a structure to combine `type` and 'fuse_code`.